examples/dlint/diagnostics.rs

```diff
@@ -9,22 +9,45 @@ pub fn display_diagnostics(
   diagnostics: &[LintDiagnostic],
   source_file: &SourceTextInfo,
   filename: &str,
+  format: Option<&str>,
+) {
+  for diagnostic in diagnostics {
+    match format {
+      Some("compact") => print_compact(filename, diagnostic),
+      _ => print_pretty(source_file, filename, diagnostic),
```

Comment:
If you apply the validator as suggested, then we should have `Some("pretty")` arm and `_ => unreachable!()`